api/src/types/request.ts

```diff
@@ -0,0 +1,46 @@
+export type Method =
+	| 'get'
+	| 'GET'
+	| 'search'
+	| 'SEARCH'
+	| 'delete'
+	| 'DELETE'
+	| 'head'
+	| 'HEAD'
+	| 'options'
+	| 'OPTIONS'
+	| 'post'
```

Comment:
Is there any way to make these case insensitive?

Reply:
unfortunately not [TS: Template Literal Types](https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html)